internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -975,15 +984,20 @@ func selectClusterNetworkType(params *models.V2ClusterUpdateParams, cluster *com
 		clusterWithNewNetworks.MachineNetworks = params.MachineNetworks
 	}
 
+	isOpenShiftVersionRecentEnough, err := common.VersionGreaterOrEqual(cluster.OpenshiftVersion, minimalOpenShiftVersionForDefaultNetworkTypeOVNKubernetes)
+	if err != nil {
+		return nil, err
```

Comment:
Here it would be nice to wrap the error `return nil, errors.Wrap(err, "failed to parse cluster OpenShift version")`